doc/classes/RigidBody2D.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RigidBody2D" inherits="PhysicsBody2D" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Rigid body 2D node.
+		Physics Body whose position is determined through physics simulation in 2D space.
```

Comment:
See all the same comments as above. "RigidBody2D" is the name of the node - don't add spaces.
Fix all instances of "If true"